corehq/apps/saved_reports/scheduled.py

```diff
@@ -88,3 +88,22 @@ def guess_reporting_minute(now=None):
             return reporting_minute
 
     raise ValueError("Couldn't guess reporting minute for time: {}".format(now))
+
+
+def _round_datetime_up_to_the_nearest_minute(dt):
+    rounded_down = datetime(dt.year, dt.month, dt.day, dt.hour, dt.minute)
+    if dt == rounded_down:
+        return rounded_down
+    else:
+        return rounded_down + timedelta(minutes=1)
+
+
+def _iter_15_minute_marks_in_range(start_datetime, end_datetime):
```

Comment:
these two functions are screaming for some unit tests

Reply:
Totally. I'll add some